File: ch12/12.2.2 MyNotes/PersistenceLayer/domain/Note.m
//
//  Note.m
//  MyNotes
//
//  Created by 关东升 on 12-9-26.
//  本书网站：http://www.51work6.com
//  智捷iOS课堂：http://www.51work6.com
//  智捷iOS课堂在线课堂：http://v.51work6.com
//  智捷iOS课堂新浪微博：http://weibo.com/u/3215753973
//  作者微博：http://weibo.com/516inc
//  官方csdn博客：http://blog.csdn.net/tonny_guan
//  QQ：1575716557 邮箱：jylong06@163.com
//

#import "Note.h"

@implementation Note

-(id)initWithDate:(NSDate*)date content:(NSString*)content {
    
    self = [super init];
    
    if (self) {
        self.date = date;
        self.content = content;
    }
    
    return self;
}


//var note = Note(date: NSDate(), content: self.txtView.text)

@end

File: ch12/12.3 MyNotes/PersistenceLayer/domain/Note.swift
//
//  Note.swift
//  MyNotes
//
//  Created by 关东升 on 2015-5-18.
//  本书网站：http://www.51work6.com
//  智捷课堂在线课堂：http://www.zhijieketang.com/
//  智捷课堂微信公共号：zhijieketang
//  作者微博：@tony_关东升
//  QQ：569418560 邮箱：eorient@sina.com
//  QQ交流群：162030268
//

import Foundation

public class Note {
    
    public var date:NSDate
    public var content:String
    
    public init(date:NSDate, content:String ) {
        self.date = date
        self.content = content
    }
}
